stash-cache-origin/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM centos:centos7
+
+RUN yum -y install http://repo.opensciencegrid.org/osg/3.4/osg-3.4-el7-release-latest.rpm \
+                   epel-release \
+                   yum-plugin-priorities && \
+    yum -y install xcache --enablerepo=osg-minefield && \
+    yum -y install supervisor cronie && \
+    yum -y install stash-origin --enablerepo=osg-development && \
+    yum clean all --enablerepo=* && rm -rf /var/cache/yum/
+
+ADD cron.d/* /etc/cron.d/
+ADD sbin/* /usr/local/sbin/
+ADD supervisord.conf /etc/
+ADD supervisord.d/* /etc/supervisord.d/
+
+ADD dummy_pod_init.sh /usr/bin/pod_init.sh
+ADD supervisord_startup.sh /usr/bin/supervisord_startup.sh
+
+CMD ["/usr/bin/supervisord_startup.sh"]
```

Comment:
I'd prefer that we defer to the base [xcache CMD](https://github.com/opensciencegrid/docker-xcache/blob/master/xcache/Dockerfile#L28) and do any implementation specific startup as their own supervisor programs, like how stash-cache init is handled: [a script](https://github.com/opensciencegrid/docker-xcache/blob/master/stash-cache/stash-cache-init.sh) configured as different [supervisor programs](https://github.com/opensciencegrid/docker-xcache/blob/master/stash-cache/supervisord.d/10-stash-cache.conf)

Reply:
Looking at the supervisor docs, I cannot see any guarantees that I can run any program before all the others. The propriety setting is just the startup ordering, but without a wait for termination.
I want hard ordering guarantees, instead.

Reply:
Yeah, sorry, I should've deleted this comment. See [this one](https://github.com/opensciencegrid/docker-xcache/pull/3#discussion_r259106286) instead